src/aleph/commands.py

```diff
@@ -109,11 +108,9 @@ def run_server_coroutine(
     # Use a try-catch-capture_exception to work with multiprocessing, see
     # https://github.com/getsentry/raven-python/issues/1110
     try:
-        loop, tasks = prepare_loop(config_values, idx=idx)
+        loop = prepare_loop(config_values)
         loop.run_until_complete(
-            asyncio.gather(
-                *tasks, run_server(host, port, shared_stats, extra_web_config)
-            )
+            asyncio.gather(run_server(host, port, shared_stats, extra_web_config))
```

Comment:
Why keep `asyncio.gather` if there is only one task ?

```suggestion
            run_server(host, port, shared_stats, extra_web_config)
```